src/librustc/dep_graph/dep_node.rs

```diff
@@ -527,6 +527,7 @@ define_dep_nodes!( <'tcx>
     [] UsedTraitImports(DefId),
     [] HasTypeckTables(DefId),
     [] ConstEval { param_env: ParamEnvAnd<'tcx, GlobalId<'tcx>> },
+    [] ConstEvalRaw { param_env: ParamEnvAnd<'tcx, GlobalId<'tcx>> },
```

Comment:
Is there a canonical place to put documentation for a query? The one for this should probably say "DO NOT USE" or so.

Reply:
added. The docs are at the query declaration, not here